.circleci/config.yml

```diff
@@ -98,7 +98,8 @@ commands:
           name: Run Integration Tests
           command: |
             kubectl wait --for=condition=ready pod/${POD_NAME} --timeout=60s -n ${NAMESPACE}
-            kubectl exec -n ${NAMESPACE} ${POD_NAME} -- ./gradlew integrationTest
+            kubectl exec -n ${NAMESPACE} ${POD_NAME} -- ./gradlew clean compileIntegrationTestJava processIntegrationTestResources
+            kubectl exec -n ${NAMESPACE} ${POD_NAME} -- ./gradlew integrationTest -Dorg.gradle.jvmargs=-Xmx2g
```

Comment:
🤮

Reply:
Yeah. Unfortunately, this split into 2 lines actually fixes our issue.